refresh-data.sh

```diff
@@ -26,7 +26,12 @@ EOF
 download_data() {
     echo 'Downloading fresh production Django database dump...'
     prod_archive="$refresh_dump_name"
-    curl -o "$prod_archive" "$CFGOV_PROD_DB_LOCATION"
+    if test -e "$prod_archive"
```

Comment:
Would you mind terribly replacing these tabs with 4 spaces? Per [the development repo](https://github.com/cfpb/development) we seem to have standardized on that.

Reply:
done. I think my editor added the tabs